### PR TITLE
Update checklist-for-installing-update-1810.md

### PR DESCRIPTION
**Pull request by russrimm**
Need a blurb to explain the SQL Server Native Client TLS 1.2 warning that blocks install during pre-req check.

### Summarize the change in the pull request title

Need to explain the SQL Server Native Client TLS 1.2 warning that stops install during pre-req check.  It's confusing to customers to have to download a new version of an old client and it's not very easy to find the links to the right version.

### CODE REVIEW

**Comment by aczechowski** [comment]
Thanks @russrimm for the contribution. I already added a similar section to the [list of prerequisite checks](https://docs.microsoft.com/sccm/core/servers/deploy/install/list-of-prerequisite-checks#sql-native-client). It makes sense to include some information in the checklist as well, but I don't like duplicating content in separate articles. Could you reword this to just a simple reference like: 

`There's now a prerequisite check for SQL Server Native Client. For more information, see the [List of prerequisite checks](/sccm/core/servers/deploy/install/list-of-prerequisite-checks#sql-native-client).`

**Comment by PRMerger7** [comment]
@russrimm : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Great, thanks @russrimm 
#sign-off 
*Once merged*, this should be live with the next daily publishing process. Right now that should be *next week* (Wednesday, 2 January) around 11:00 Pacific time. (There's a publishing freeze for the holidays.) No further action needed from you.
